packages/browser/src/core/stats/remote-metrics.ts

```diff
@@ -14,9 +14,37 @@ export interface MetricsOptions {
  */
 type RemoteMetric = {
   type: 'Counter'
-  tags: Record<string, string>
   metric: string
-  value: number
+  value: 1
+  tags: {
+    library: string
+    library_version: string
+    [key: string]: string
+  }
+}
+
+const createRemoteMetric = (
+  metric: string,
+  tags: string[],
+  versionType: 'web' | 'npm'
+): RemoteMetric => {
+  const formattedTags = tags.reduce((acc, t) => {
+    const [k, v] = t.split(':')
+    acc[k] = v
+    return acc
+  }, {} as Record<string, string>)
```

Comment:
This could be condensed to `Object.fromEntries(tags.map(el => el.split(":")))` -- but dang browser support =/